ansible_base/authentication/utils/authentication.py

```diff
@@ -145,11 +145,7 @@ def _handle_no_merge_strategy(uid: str, authenticator: Authenticator) -> str:
     if AuthenticatorUser.objects.filter(Q(uid=uid) | Q(user__username=uid)).exists():
         # Some other provider is providing this username so we need to create our own username
         new_username = get_local_username({'username': uid})
-        logger.info(
-            f'Authenticator {authenticator.name} wants to authenticate {uid} but that'
-            f' username is already in use by another authenticator,'
-            f' the user from this authenticator will be {new_username}'
-        )
+        logger.warning(f"User {uid} is not found in the database, creating a new user with username {new_username}")
```

Comment:
This warning doesn't make sense to me.  It appears that this branch is for uid found in db (but w/ a different authenticator) not that it's missing from the db.

Reply:
Great call,
I will update to this log - 
```
logger.warning(f"User {uid} is already associated with an existing user, creating a new user with username {new_username}")
```